src/Kafka/ConsumerGroups/components/ConsumerGroupResetOffset.tsx

```diff
@@ -0,0 +1,274 @@
+              <StackItem>
+                <TableComposable
+                  aria-label="Selectable Table"
+                  className="consumer-table"
+                >
+                  <Thead>
+                    <Tr>
+                      <Th
+                        select={{
+                          isSelected: isSelected,
+                          onSelect: (_event, isSelected) =>
+                            onSelectAll(isSelected),
+                        }}
+                      />
+                      <Th>{tableColumns.partition}</Th>
+                      <Th>{tableColumns.clientId}</Th>
+                      <Th>{tableColumns.current_offset}</Th>
+                      <Th>{tableColumns.log_end_offset}</Th>
+                      <Th>{tableColumns.offset_lag}</Th>
+                      <Th>{tableColumns.new_offset}</Th>
+                    </Tr>
+                  </Thead>
+                  <Tbody>
+                    {consumers.map((consumer, index) => {
+                      return (
+                        <Tr key={index}>
+                          <Td
+                            select={{
+                              rowIndex: index,
+                              isSelected: isSelected,
+                              onSelect: (_event) =>
+                                onSelectRow(index, isSelected),
+                            }}
+                          />
+                          <Td dataLabel={tableColumns.partition}>
+                            {consumer.partition}
+                          </Td>
+                          <Td dataLabel={tableColumns.clientId}>
+                            {String(consumer.groupId)}
+                            {String(consumer.memberId)}
+                          </Td>
+                          <Td dataLabel={tableColumns.current_offset}>
+                            {consumer.offset}
+                          </Td>
+                          <Td dataLabel={tableColumns.log_end_offset}>
+                            {consumer.logEndOffset}
+                          </Td>
+                          <Td dataLabel={tableColumns.offset_lag}>
+                            {consumer.lag}
+                          </Td>
+                          <Td dataLabel={tableColumns.offset_lag}>
+                            {consumer.selected && selectedOffset
+                              ? selectedOffset === "absolute"
+                                ? customOffsetValue
+                                : selectedOffset
+                              : "-"}
+                          </Td>
+                        </Tr>
+                      );
+                    })}
+                  </Tbody>
+                </TableComposable>
+              </StackItem>
+              <StackItem>
+                <Checkbox
+                  label={t("consumerGroup.reset_offset_accept")}
+                  aria-label="uncontrolled checkbox example"
```

Comment:
`aria-label` should not be used here. The `<Checkbox>` automatically handles associating the label with the input using the `id`/`for` method.